systemlock-0.1/debian/control

```diff
@@ -10,3 +10,8 @@ Package: systemlock
 Architecture: all
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: Interface for ofris.
```

Comment:
While you are revising this: What is "ofris" btw?

Reply:
Ofris its given by @pranav1698 before hand.

Reply:
I am aware it is not your line.

Reply:
So, ??